bundles/remote_services/remote_service_admin_shm_v2/shm_pool/src/shm_cache.c

```diff
@@ -240,7 +240,7 @@ static void * shmCache_WatcherThread(void *data) {
         for (int i = 0; i < size; ++i) {
             shm_cache_block_t *shmBlock = celix_arrayList_get(evictedBlocks, i);
             celix_longHashMap_remove(shmCache->shmCacheBlocks, shmBlock->shmId);
-            fprintf(stdout, "Shm cache: Shm(%d) has been closed, colse its cache.\n", shmBlock->shmId);
+            fprintf(stdout, "Shm cache: Shm(%d) has been closed, close its cache.\n", shmBlock->shmId);
```

Comment:
> New code should not use raw printf. Without access to bundle/framework context, we can still use logging macro, which defaults to no output. Or shall we use syslog instead of printf? I suggest a discussion with @pnoltes

Good point. Indeed fprintf should ideally not be used, to be honest I am not sure what the best solution is for libraries which do not have access to the Celix framework.